tools/kagami/src/main.rs

```diff
@@ -713,18 +717,44 @@ mod validator {
     impl<T: Write> RunArgs<T> for Args {
         fn run(self, writer: &mut BufWriter<T>) -> Outcome {
             writer
-                .write_all(&construct_validator()?)
+                .write_all(&construct_validator(None)?)
                 .wrap_err("Failed to write wasm validator into the buffer.")
         }
     }
 
-    pub fn construct_validator() -> color_eyre::Result<Vec<u8>> {
+    pub fn construct_validator(
+        swarm_build_dir: Option<&AbsolutePath>,
+    ) -> color_eyre::Result<Vec<u8>> {
         let build_dir = tempfile::tempdir()
             .wrap_err("Failed to create temp dir for runtime validator output")?;
+        let source_dir = tempfile::tempdir()
+            .wrap_err("Failed to create temp dir for default runtime validator sources")?;
+
+        // If `CARGO_MANIFEST_DIR` is set, it means the `kagami` binary is run from inside Iroha repo,
```

Comment:
```suggestion
        // If `CARGO_MANIFEST_DIR` is set, it means the `kagami` binary is run with `cargo run`,
```

I am not sure, but when we tested this with @timofeevmd, it turned out that even running a compiled `kagami` binary from the root of the repo doesn't work, because `CARGO_MANIFEST_DIR` env variable is not set.

Reply:
Changed the wording, but the new version also works with compiled standalone, added a little CI check for that too.